lib/python/grassdb/checks.py

```diff
@@ -86,6 +86,15 @@ def get_mapset_owner(mapset_path):
         return None
 
 
+def get_gislock_owner(lock_path):
+    """Returns gislock owner name or None if owner name unknown"""
+    try:
+        path = Path(lock_path)
+        return path.owner()
+    except KeyError:
+        return None
```

Comment:
I think the best thing to do with this function is to turn it into `get_mapset_lock_info(mapset_path)`. It can get also the timestamp (and there is one other thing which this could get but that's too much for this PR). Given that there will/might be extensions and for the lack of better ideas, the return type should be dict with keys owner and datetime.

Actually, let's add also filename as another key which will remove the need to call `get_lockfile_if_present()` and you can use the nicer `is_mapset_locked()`.